discovery-infra/tests/config/__init__.py

```diff
@@ -0,0 +1,9 @@
+from ._global_configs import ClusterConfig, NodesConfig
```

Comment:
I really don't like this way. I don't think it is somehow required.
@osherdp  @YuviGold what do you think?

Reply:
@tsorya What way?

Reply:
If you are talking about the private modules:

Because the library and usage of the library are on the same repo, we want to allow only specific object to be exported, for the same reason we need to use private/protected properties in our classes.
All public can cause us a lot of issues that will cost us time and effort.

Reply:
@tsorya please be more specific next time
If talking about the private modules, then I agree. @eliorerz, no one does that on Python and it has different rules and conventions than golan/Java/C# etc.

Reply:
i meant that i don't like adding these __init__ file and using private modules too.

Reply:
@osherdp @tsorya  Using protected modules is not that common but it indeed in use (e.g. pytest, flake8)
It is more needed in python (and not on other languages) because of the "freedom" that the developer has, more boundaries are needed.
Anyway, I think it will prevent us for making more mistakes as we did before (especially while the infra and its usage are on the same place).
I think if you don't have a major objection to this we can leave it and refactor it back is pretty easy, else I can just remove it (but again I think it's wrong).

Reply:
Sorry, I do object this
As it's a simple change, let's change this now to stay coherent to our repo's conventions

Reply:
About the init usage, I don't have any preferences

Reply:
@osherdp It just the first step (and first task), but a lot of this repo's conventions are going to change (during MGMT-4975).

Reply:
Sorry, I just can't approve this specific convention change.
No hurt feelings, though :yellow_heart: